scripts/uberenv/packages/umpire/package.py

```diff
@@ -276,6 +277,18 @@ def hostconfig(self, spec, prefix, py_site_pkgs_dir=None):
             cfg.write(cmake_cache_entry("CMAKE_CXX_FLAGS_RELWITHDEBINFO", reldebinf_flags))
             cfg.write(cmake_cache_entry("CMAKE_CXX_FLAGS_DEBUG", debug_flags))
 
+        #Configuration to enable developer benchmarks (i.e. No-Op)
+        if "+dev_benchmarks" in spec:
+            cfg.write("#------------------{0}\n".format("-" * 60))
+            cfg.write("# Developer Benchmarks\n")
+            cfg.write("#------------------{0}\n\n".format("-" * 60))
+            
+            cfg.write(cmake_cache_option("ENABLE_DEVELOPER_BENCHMARKS", True))
+            cfg.write(cmake_cache_string("CMAKE_BUILD_TYPE", "Release"))
```

Comment:
Don't force this here, you should set it in the SPEC variable of the CI configs. It will be added to the host-config file by this line: https://github.com/LLNL/Umpire/blob/d21ae5e42ef4440296232112ae488e0af4ca7265/scripts/uberenv/packages/umpire/package.py#L209